INSTALL.md

```diff
@@ -7,6 +7,7 @@ This file contains information on how to configure and run the BPDM applications
 BPDM Pool is a SpringBoot Kotlin software project managed by Maven.
 
 The project can be run with the following command: `mvn clean spring-boot:run`
+The command should be run from the root poom file
```

Comment:
The project can't be run with that command from the parent pom. You would need to specify the module as a target. We can maybe tell here that we can build this whole project from the parent pom and then depending on which application we want to run we need to specify the respective module